src/runtime/components/forms/Toggle.vue

```diff
@@ -51,7 +53,15 @@ export default defineComponent({
       default: null
     },
     modelValue: {
-      type: Boolean,
+      type: AnyType,
```

Comment:
I think it would be better to declare the array directly here for consistency.
```suggestion
      type: [Boolean, String, Number, Object],
```